### DISCUSSION TITLE
Poor Performance on Integrated Graphics

### DISCUSSION THREAD

**Discussion by aer0ace** [wonderunit/storyboarder#2445] [General]
I get really poor performance on brush strokes when using a core i3 machine (or below) with no hardware acceleration.
But, on the same machine, I have Affinity Photo, Clip Studio Paint, and Open Toonz installed, and none of those applications has problems with brush stroke performance. Will this be addressed in the new version of Storyboarder?

**Comment by RannyBergamotte** [comment]
That's cause storyboarder runs electron: long story short, that basically means that the ui is rendered in a web browser like chrome, so it will hog memory and takes a lot of resources to run a browser. The other programs, are probably written in a more sensible way. For example OpenToonz is written in C++ which runs machine level code, so it's soooo much faster. Developers love electron though cause its easy to use (cause you're writing a website, which also means write once works everywhere (web, Windows, MacOS, Linux)), while for C++ or other low level languages you would have to compile for each OS.

Given the state of this repo, doubt it'll ever get addressed lol. I'll join you in the suffering of electron.
